app/cli/cmd/root.go

```diff
@@ -37,6 +37,8 @@ var (
 	flagOutputFormat string
 	actionOpts       *action.ActionsOpts
 	logger           zerolog.Logger
+	defaultCpAPI     = "api.cp.chainloop.dev:443"
```

Comment:
since CAS and CP are acronyms, I'd update them. 

`defaultCpAPI` => `defaultCPAPI`
`defaultCasAPI` => `defaultCASAPI`